_scripts/compile-backend-ts-services.sh

```diff
@@ -6,7 +6,7 @@ fi
 # 1. Get the list of all modified and dependent services.
 echo "checking for modified services..."
 # Cleans up yarn's output by removing everything before the package name, the first and the last line.
```

Comment:
Q: Is the comment still accurate [if we're no longer removing the first line of 'yarn workspaces list' output]?